styles.css

```diff
@@ -0,0 +1,147 @@
+body {
+  background-color: #eee;
+  font-family: 'lato', sans-serif;
+}
+
+.logo {
+  width: 60vw;
+}
+
+#header {
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
```

Comment:
- **[OPTIONAL]** - Give your Header a Background Color and also a z-index to make it show on other elements.